src/stylesheets/components/_card.scss

```diff
@@ -31,127 +56,275 @@
 // Header
 // ---------------------------------
 .usa-card__header {
-  @include u-padding-bottom($theme-card-padding / 2);
-  @include u-padding-top($theme-card-padding);
-  @include u-padding-x($theme-card-padding);
+  @include u-padding-bottom($theme-card-padding-y / 2);
+  @include u-padding-top($theme-card-padding-perimeter);
+  @include u-padding-x($theme-card-padding-perimeter);
+
+  &:last-child {
+    @include u-padding-bottom($theme-card-padding-perimeter);
+  }
 }
 
 .usa-card__heading {
-  @include typeset($theme-card-header-font-family, "lg", 2);
+  @include typeset($theme-card-header-typeset);
   @include u-margin(0);
 }
 
+// Media
+// ---------------------------------
+.usa-card__media {
```

Comment:
Image is popping out in IE11.

![image](https://user-images.githubusercontent.com/3385219/80509481-1a92e180-893f-11ea-99de-eefc339f1376.png)